### PR TITLE
pkg/trace/{agent,stats}: add tag for synthetic data

### PR DESCRIPTION
**Pull request by kristinekunapuli**
### What does this PR do?

Adds the tag "synthetic:true" when _dd.origin has a value starting with "synthetics". This is to allow independent aggregation of metrics generated from stats payloads. 

### Motivation

What inspired you to submit this pull request?

### Additional Notes

Anything else we should know when reviewing?

### Describe your test plan

Write there any instructions and details you may have to test your PR.

### CODE REVIEW

File: pkg/trace/stats/aggregation.go [hunk 1]
```diff
@@ -10,6 +10,8 @@ const (
 	tagHostname   = "_dd.hostname"
 	tagStatusCode = "http.status_code"
 	tagVersion    = "version"
+	tagOrigin     = "_dd.origin"
+	tagSynthetic  = "synthetic"
```
Comment (bmermet) [inline pkg/trace/stats/aggregation.go:14]: Why is the tag called syntheticrather than synthetics like in _dd.origin ?

Comment (kristinekunapuli) [reply pkg/trace/stats/aggregation.go:14]: That's my mistake. I'll fix it. Thank you!

**Review by bmermet — APPROVED** [review]
LGTM.

**Comment by gbbr** [comment]
Please don't merge things during the freeze 🙂 🥶 🤭

**Pull request merged**